include/deal.II/base/tensor_base.h

```diff
@@ -1,6 +1,7 @@
 #ifndef dealii__tensor_base_h
 #define dealii__tensor_base_h
 #warning This file is deprecated. Use <deal.II/base/tensor.h> instead.
-#endif
 
 #  include <deal.II/base/tensor.h>
+
+#endif
```

Comment:
Sure, the same would be true for all `include/deal.II/base/std_cxx1x/*.h` files. 
I just consider it good style to have these guards in all header files.